Main.java

```diff
@@ -1,78 +1,195 @@
+    	 float length = scan.nextFloat();
+    	 
+    	 Square sqr = new Square(length);
+    	 System.out.printf("The area of a square with side: %.2f is %.2f",length, sqr.area());
+    	 System.out.println(" ");
+    	 System.out.printf("The perimeter of a square with side: %.2f is %.2f",length, sqr.perimeter());
+    	 break;
+    	 
+       //Rectangle case
+    case 'R':
+    	System.out.println("This programs calculates the area and perimeter of the Rectangle");
+    	System.out.println("Input length");
+    	float l1 = scan.nextFloat();
+    	System.out.println("Insert breadth");
+    	float b2 = scan.nextFloat();
+    	
+    	Rectangle rec = new Rectangle(l1,b2);
+   	 	System.out.printf("The area of a rectangle with length: %.2f and breadth: %.2f is %.2f",l1,b2,rec.area());
+   	 	System.out.println("");
+   	 	System.out.printf("The perimeter of a rectangle with length: %.2f and breadth: %.2f is %.2f", l1,b2,rec.perimeter());
+      break;
+
+
+     }
+  }
+}
 
-  
-    System.out.println("Input the dimension of the first pair of your Parallelogram");
-    q.setLength(scan.nextFloat());
-    System.out.println("Input the second pair of your Parallelogram");
-    q.setBreadth(scan.nextFloat());
+//Create class for Rhombus
+class Rhombus extends Quadilateral{
+  private float diagonal1;
+  private float diagonal2;
+  private float side;
+  public Rhombus(float d1,float d2, float rs){
+    diagonal1 = d1;
+    diagonal2 = d2;
+    side = rs;
     
-    System.out.printf("The area of the parallelogram with base %.2f and height %.2f is %.2f", pBase, pHeight, p.Area());
-
-    System.out.println("");
-    System.out.printf("The perimeter of the parallelogram with First Pair: %.2f and SecondPair: %.2f is %.2f", q.getLength(), q.getBreadth(), q.calculatePerimeter());
-    System.out.println("");
-    System.out.println("");
-    System.out.println("");
-
-    //this should be in the Trapezoid case
-     System.out.println("Calculation for a Trapezoid");
-     System.out.println("Input first side");
-     float a = scan.nextFloat();
-     System.out.println("Input second side");
-     float b = scan.nextFloat();
-     System.out.println("Input third side");
-     float c = scan.nextFloat();
-     System.out.println("Input fourth side");
-     float d = scan.nextFloat();
-     Trapezoid t = new Trapezoid(a, b, c, d);
-    System.out.printf("The perimeter of the Trapezoid with First Side: %.2f, Second Side: %.2f, Third Side: %.2f, and Fourth Side: %.2f is %.2f", a, b, c, d, t.calculatePerimeter()); 
+
   }
+public float area(){
+  return (diagonal1 * diagonal2)/2;
+  }
+public float perimeter() {
+	return 4*side;
+	}
 }
 
-//Create class for Rhombus
 
 
 //Create class for square
-
+class Square extends Quadilateral{
+	private float length;
+	public Square(float sl) {
+	length = sl;	
+	}
+	
+	public float area() {
+		return (float) Math.pow(length,2);
+	}
+	
+	public float perimeter() {
+		return 4*(length);
```

Comment:
correct but you should have put the calculation in the quadrilateral class because it is the same with the rhombus area and call it something like: area_of_all_equal_sides